googleapiclient/http.py

```diff
@@ -1394,6 +1394,15 @@ def execute(self, http=None):
     if http is None:
       raise ValueError("Missing a valid http object.")
 
+    # Special case for OAuth2Credentials-style objects which have not yet been
+    # refreshed with an initial access_token.
+    if http is not None and hasattr(http.request, 'credentials'):
+      creds = http.request.credentials
+      if hasattr(creds, 'access_token') and not creds.access_token:
+        LOGGER.info('Attempting refresh to obtain '
```

Comment:
Please avoid implicit string concatenation whenever you can - does this string really not fit in eighty columns? If no, line-wrap after the opening parenthesis?